onebusaway-android/build.gradle

```diff
@@ -298,5 +298,6 @@ dependencies {
     // Google Play Services Maps (only for Google flavor)
     googleCompile 'com.google.android.gms:play-services-maps:8.3.0'
     // Amazon Maps (only for Amazon flavor)
-    amazonCompile 'com.amazon.android:amazon-maps-api:2.0'
+    amazonCompile('com.amazon.android:amazon-maps-api:2.0')
+    compile 'edu.usf.cutr:open311client:1.0.6'
```

Comment:
Could you please move this open311 library above the libraries that are specific to a build variant?  I'd like to keep all the ones required for all variants in the same section of `build.gradle`.